app/mailers/submission_mailer.rb

```diff
@@ -0,0 +1,37 @@
+require "#{Rails.root}/app/helpers/application_helper.rb"
+include ApplicationHelper
+
+class SubmissionMailer < ApplicationMailer
+
+  def new_submission_confirmation_email(submission)
+    @submission = submission
+
+    @form_name = @submission.form_name
+    if @form_name.downcase.include?("ask")
+      @form_contact = Organization.current_organization.ask_form_contact
+    elsif @form_name.downcase.include?("offer")
+      @form_contact = Organization.current_organization.offer_form_contact
+    elsif @form_name.downcase.include?("community_resource")
+      @form_contact = Organization.current_organization.community_resources_contact
+    end
+
+    @person = @submission.person
+    @locale = @person.preferred_locale || 'en'
```

Comment:
TODO - use this info to call different templates based on language preference (and using Mobility fallbacks?)

Reply:
(Oops. I split the addition of this field into another PR.)